database/resources/events.json

```
@@ -0,0 +1,24 @@
+[
+  {
+    "name": "Hackathon",
+    "description": "A hackathon is an event that brings together experts and creates a collaborative environment for solving a certain problem..",
```

[Proofreading] Consider describing what the article teaches as opposed to defining the term. It'll encourage developers to read the article. 
```suggestion
    "description": "A hackathon is an event that brings together experts and creates a collaborative environment for solving a certain problem..",
```